Storage/SQL/BrowserTable.swift

```diff
@@ -240,17 +243,46 @@ public class BrowserTable: Table {
         }
 
         log.debug("Updating browser tables from \(from) to \(to).")
-        if from < 4 {
+        if from < 4 || to < from {
             return drop(db) && create(db, version: to)
         }
 
+        // Create the queue table
         if from < 5 {
-            let queue =
-            "CREATE TABLE IF NOT EXISTS \(TableQueuedTabs) (" +
-            "url TEXT NOT NULL UNIQUE, " +
-            "title TEXT" +
-            ") "
-            if !self.run(db, queries: [queue]) {
+            if !self.run(db, queries: [CreateQueueTable]) {
+                return false
+            }
+        }
+
+        // Moved the date column from the favicons table to the faviconSites table.
+        if (from < 6) {
```

Comment:
I'd like to make something a little nicer, but I'm fine getting there incrementally.